action.yml

```diff
@@ -2,11 +2,15 @@ name: 'Create Monorepo Release PR'
 description: 'Update the versions of monorepo packages and create a release PR.'
 
 inputs:
+  initial-release:
```

Comment:
Do we really need this input :thinking: ? I thought we'd be able to automatically detect a lack of tags, so we'd be able to detect whether it's the initial release or not.

Reply:
Oh we can definitely detect a lack of tags, but the input is the best way I could think of to determine whether that's expected or not.

You'll notice in the example consumer workflow in the readme that it uses `actions/checkout@v2` with `fetch-depth: 0`, which makes it fetch the entire history. By default, the `checkout` action will only fetch a single commit. I ran into exactly this problem while developing this action, and I thought it safest to assume that something is wrong if there are no tags. The `initial-release` input exists to let the consumer say "I know that there are no tags, so don't worry about it".

Reply:
Ah right, the fetch-depth thing, I forgot about that.

Would it be possible to detect that circumstance instead (e.g. https://stackoverflow.com/questions/37531605/how-to-test-if-git-repository-is-shallow)? If we could detect when we had an incomplete git history, we could error early. This would make the action safer to use, and remove the necessity for this input.

Reply:
Following the suggestion in that Stackoverflow question, I completely revised the implementation in 779d26f.